pkg/storegateway/storepb/types.proto

```diff
@@ -31,7 +31,7 @@ message Chunk {
 }
 
 message Series {
-  repeated Label labels      = 1 [(gogoproto.nullable) = false, (gogoproto.customtype) = "github.com/grafana/mimir/pkg/storegateway/labelpb.ZLabel"];
+  repeated cortexpb.LabelPair labels      = 1 [(gogoproto.nullable) = false, (gogoproto.customtype) = "github.com/grafana/mimir/pkg/mimirpb.LabelAdapter"];
```

Comment:
Why do we need the change here to `cortexpb.LabelPair`?

Reply:
Because I deleted Label.

Reply:
Oh it was in types.proto. Sorry I missed that one.

Reply:
So the proto will unmarshal even if the type name is different? In that case, should we create a task to rename `cortexpb` to `mimirpb`?

Reply:
I added more info to the PR description.